test/check-subscriptions

```diff
@@ -476,6 +444,7 @@ class TestSubscriptions(SubscriptionsCase):
 
 
 class TestSubscriptionsPackages(SubscriptionsCase, PackageCase):
+    @skipUnlessDistroFamily("rhel", "Insights support is specific to RHEL")
```

Comment:
there is no need to disable this test on non-RHEL OSes: the test itself creates a fake `insights-client` package which is added to the local repositories of the system (so PackageKit can install it)

Reply:
The fake `insights-client` package is created, but this package does not contain any `insights-client` binary. It is reason, why this test fails on non-rhel distribution.

Reply:
`TestSubscriptionsPackages.testMissingPackages` worked fine on non-RHEL distros already, and still works fine; the test actually relies on the fake `insights-client` package, as it checks that that package is installed via packagekit